src/lib/moment/momentUtils.ts

```diff
@@ -5,3 +5,14 @@ import moment from 'moment';
 export const formatUTCDateStringToLocal = (dateStringUTC: string): string => {
   return moment(dateStringUTC).local().format('YYYY-MM-DD HH:mm:ss');
 };
+
+//takes in a string of a date time YYYY-MM-DD format. Returns true if its before the current month.
+export const lessThanCurrentMonth = (dateString: string): boolean => {
+  let lastDate = moment(dateString);
+  let lastMonth = lastDate.get('month');
```

Comment:
`lastMonth` and `lastDate` aren't good variable names here since they hint that it's from a month ago